src/quantum/widget.jsx

```diff
@@ -35,14 +36,14 @@ export default class Widget extends React.Component {
     );
     let $secondTitle;
 
-    if (this.props.secondTitle) {
-      if (this.props.secondLink) {
-        $secondTitle = (
-          <Link to={this.props.secondLink}>{this.props.secondTitle}</Link>
-        );
-      } else {
-        $secondTitle = this.props.secondTitle;
-      }
+    if (this.props.secondLink) {
+      $secondTitle = (
+        <Link to={this.props.secondLink} title="show details">
+          <DetailsIcon />
+        </Link>
+      );
+    } else {
+      $secondTitle = '';
```

Comment:
Use icon instead of word

Reply:
I'm curious why you're using a $ in front of secondTitle.

Reply:
That is curious; I did not even notice!  I was simply using the variable that was used before.  I will fix this.